### ISSUE TITLE
Pre trained vectors

### ISSUE THREAD

**Issue by karanchahal** [ruotianluo/ImageCaptioning.pytorch#37]
Hi I'm making my own version if a image captioning model, I haven't gone through your code in detail yet. I was wondering if you have used pre-trained word vectors for this task or just used an one hot encoding representation ?
And do you think the use of pre-trained word vectors make a substantial impact on training time and accuracy ?

**Comment by ruotianluo** [comment]
Ive never tried, but it has been claimed on some paper that pretrained word vectors don't help much.

**Comment by karanchahal** [comment]
Transfer learning in NLP doesn't seem to have the same impact as that of images it seems like. Any good papers for transferring knowledge of pretrained nlp systems you recommend ?
